sourced/ml/transformers/repo2df.py

```diff
@@ -9,6 +9,14 @@ def __init__(self, extractors, **kwargs):
         self.extractors = extractors
 
     def __call__(self, rows):
+        for extractor in self.extractors:
```

Comment:
Create another class `Repo2QuantizedDocFreq` and put it there.

Reply:
Thanks for reviewing Egor! adding another class here would make me add few lines in apollo source2bags() but I can do it like this yes